topology/src/main/resources/proto/requests.proto

```diff
@@ -7,21 +7,25 @@ option java_package = "io.camunda.zeebe.topology.protocol";
 
 message AddMembersRequest {
   repeated string memberIds = 1;
+  bool dryRun = 2;
```

Comment:
:thought_balloon: Would it be useful to define a general reusable `Options`? In future, if we want to add more options then it will be easier. However, right now I can't think of any new flags that will be useful in the future.
```
message Options {
    bool dryRun = 1;
    // in future
   type someOtherFlag = 2;
```